### ISSUE TITLE
Enforce constants to be SCREAMING_SNAKE_CASE

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1885]
# Rule request

## Thesis

According to PEP-8:

> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.

Examples:

```python
# good
SOMETHING = 'hello'

def hello():
  # good, not a global var
  something = 'hello'

# good, not a constant
something = User()

# bad
something = 'hello'
```

PyLint has `C0103` rule to validate it.

## Reasoning

Follow PEP-8

**Comment by skv0zsneg** [comment]
Hi!
I solved one of the issue (#1600) with similar problem. If you don't mind, I would like to work on this 😄

**Comment by sobolevn** [comment]
please, check that `ruff` does not have this.

**Comment by skv0zsneg** [comment]
Cannot find here https://docs.astral.sh/ruff/rules/  anything similar

**Comment by skv0zsneg** [comment]
@sobolevn should I search somewhere else?
